dts/arm/nordic/nrf54h20_eng0_cpuapp.dtsi

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54h20_eng0.dtsi>
+
+cpu: &cpuapp {};
+systick: &cpuapp_systick {};
+nvic: &cpuapp_nvic {};
+
+/delete-node/ &cpuppr;
+/delete-node/ &cpurad;
+/delete-node/ &cpurad_peripherals;
+/delete-node/ &cpurad_ppb;
+/delete-node/ &cpurad_ram0;
+
+/ {
+	soc {
+		compatible = "simple-bus";
```

Comment:
can simple-bus, ranges be moved to the global dt file?

Reply:
The idea with setting `simple-bus` and `ranges` here is to indicate that this CPU is connected to this bus, while other CPUs may or may not be. In the global dt file, they should appear as "indirect buses", to borrow a system DT term.

Reply:
ok, got it